### DISCUSSION TITLE
question about kobold standalone executable. does it spawn directories somewhere?

### DISCUSSION THREAD

**Discussion by joshgura** [LostRuins/koboldcpp#1121] [Q&A] [answered]
I am using the standalone executable launcher for kobold and everything is working. i'm just wondering if the standalone creates temp files or logs, etc somewhere on my home folder. for logs, media such as wav files that were passed from tts, and anything else.

reason i ask is i'm using alltalk and the wav outputs aren't getting saved by alltalk so i suspect kobold is receiving the files and either saving them somewhere or sending them to dev null after playing them.

**Comment by LostRuins** [comment, marked as answer]
The standalone exe creates a temporary pyinstaller folder in your temp dir, it is automatically removed when the program exits. However, this is only used for the dll libraries to let kobold function. All data such as TTS voice clips and images are not stored anywhere but within the browser. They are encoded/decoded at runtime and sent over the API within a request.

For AllTalk, you'd have to check with the dev if the responses are saved elsewhere. But they are not saved on the Kobold side.
